README.md

```diff
@@ -26,12 +26,16 @@ i.e.
 #### Tests
 
 - How do you run tests?
+
 no test currently exist
+
 - Any tests of note?
+
 no
+
 - Describe any tests that you did not complete, skipped, etc
 Did not complete any tests
 
 #### UML
 
-No current response, did not get to deploying on heroku
\ No newline at end of file
+https://lab-4-api-server.herokuapp.com
```

Comment:
Thanks for getting that heroku link in there. I would love to see tests added to this!